Bottom: cd37e742d89248b53973fa9fdf594fe8d07ab8b5
Top:    59f4f7886304ee30c5b7827c6e8eb8d6aed29100
Author: scott snyder <sss@karma>
Date:   2020-02-03 00:34:13 -0500

Refresh of pydepr

---

diff --git a/DDG4/tpython/DDPython.cpp b/DDG4/tpython/DDPython.cpp
index 9a627860..4dca934a 100644
--- a/DDG4/tpython/DDPython.cpp
+++ b/DDG4/tpython/DDPython.cpp
@@ -305,7 +305,7 @@ void DDPython::prompt()  const   {
 void DDPython::afterFork()  const  {
   if ( ::Py_IsInitialized() ) {
     cout << "[INFO] Re-init python after forking....." << endl;
-    ::PyOS_AfterFork();  
+    ::PyOS_AfterFork_Child();  
     ::PyEval_InitThreads();
     ::PyEval_ReleaseLock();
   }
